docs/dg/dev/internationalization-and-multi-store/add-new-store-in-multi-db-setup.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Adding new store in multi-db setup"
+description: Adding and deploying a new store in multi-db store setup requires additional steps and preparation. This guideline contains all the best practices you need to know.
+last_updated: Dec 02, 2024
+template: howto-guide-template
+originalLink: 
+originalArticleId: 
+redirect_from:
+---
+
+Setting up a new store in an existing multi-DB environment requires a well-prepared plan, to avoid disrupting the data and operations of existing stores. This guide provides a detailed procedure to release a new store within a region that already hosts other stores, ensuring a non-destructive deployment.
+
+## Initial planning and best practices
+
+### Clear Roadmap
+It is good to have overall plan, detailing all stores that will be added in the future. This can impact not only database structure and configurations, but overall decisions on how to approach the rollout, making sure it is cost-efficient over time, on all ends.
+
+### Backup strategy
+Always have a backup plan ready in case of issues during the deployment. This includes not only database backups but also considerations on all points you will find below, including the business side.
+
+### Environment Preparation
+Prepare the development, staging, and production environments for a new store rollout. Make sure you don’t have additional functionality to be released on top or parallel development. This involves ensuring that teams are prepared and stakeholders are aware of the procedure.
```

Comment:
> Prepare the development, staging, and production environments for a new store rollout
Huge part of our customers have only staging and production by contract, I think it worth to mentioned somehow

Reply:
done, thank you!